src/pybind/mgr/orchestrator_cli/run-tox.sh

```diff
@@ -23,10 +30,17 @@ then
 fi
 
 if [ "$WITH_PYTHON2" = "ON" ]; then
-  ENV_LIST+="py27"
+  ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "ON" ]; then
-  ENV_LIST+=",py3"
+if [ "$WITH_PYTHON3" = "3" ]; then
+  ENV_LIST+="py3,"
 fi
+ENV_LIST=$(echo "$ENV_LIST" | sed -e 's/,$//')
```

Comment:
This line is not required anymore

Reply:
Damn, overlooked that one.

This can be fixed in the next round of changes in master - i.e. when we combine all of these near-duplicate scripts into a single script.

Reply:
done in #28002